app/components/Card/CardBack.tsx

```diff
@@ -85,14 +85,14 @@ export default function CardBack({
 
 const responsiveValues = {
   carousel: {
-    size: { md: "back_md", "2xl": "back_lg" },
+    size: { xs: "back_xs", sm: "back_sm", md: "back_md", "2xl": "back_lg" },
```

Comment:
ChakraUI uses "base" keyword for 0px breakpoint, that's why it didn't work for mobile view. 
```suggestion
    size: { base: "back_xs", sm: "back_sm", md: "back_md", "2xl": "back_lg" },
```
Great job adding 2 more sizes for the machine cards! Though for that to properly work you will also have to add the corresponding "base" and "sm" sizing to lines 90-97.  Btw, it looks like I forgot to change "md" breakpoint to "base" on those lines; so that contributed to the cards not displaying properly on screens smaller than 768px before you introduced new cards.

I believe that in Figma there are only 2 sizes of cards, one is used for mobile and the other for tablet and desktop. I think having 4 sizes might be an overkill for the carousel. If the card sizes that I've set up still seem too big, let's just switch to the new sizes that you created. What do you think?

Reply:
I now changed all of the first sizes to "base", even including the ones that previously were "md".